semantic/src/definitions/collections/menu.less

```diff
@@ -1293,6 +1293,11 @@ Floated Menu / Item
   border-color: @blue !important;
   color: @blue !important;
 }
+.ui.menu .blueDark.active.item,
+.ui.blueDark.menu .active.item {
+  border-color: @blueDark !important;
+  color: @ink !important;
```

Comment:
https://universedev.atlassian.net/browse/WEB-12687 - Choosing to add this as a sort of render mode, since I am not sure whether we want to have all the menu items to behave the same way with only bottom border showing colour, while the text colour is set to ink/certain font-weight